inspirehep/config.py

```diff
@@ -40,6 +40,8 @@ def _(x):
 # =====
 DEBUG_TB_INTERCEPT_REDIRECTS = False
 SERVER_NAME = "localhost:5000"
+ASSETS_DEBUG = True
+DEBUG = True
 
```

Comment:
These config variables should be in your personal var/inspirehep-instance/inspirehep.cfg